lib/connextclient/ConnextClient.ts

```diff
@@ -900,6 +901,21 @@ class ConnextClient extends SwapClient {
     return parseInt(blockNumberResponse.result, 16);
   }
 
+  private getGasPrice = async () => {
```

Comment:
A comment here would be helpful too in regards to what exactly this returns - is this getting the most recent average gas price? Or is this derived from some fee estimation logic built into geth?

Reply:
Shouldn't break anything, here a heads up for this upcoming change in geth how gas prices are calculated (moving from the current auction model/fee market to a moving average): https://github.com/ethereum/go-ethereum/pull/21971 

thanks for pointing me at it @michael1011

Reply:
The method call Karl is using returns data from GETHs internal fee estimation logic.

What Kilian mentioned is EIP-1559 which will be a breaking change in the way Ethereum handles gas prices and eventually be activated in a hardfork. But that won't matter for us since the REST endpoints of GETH won't change because of that.

Reply:
Comment added.